Update README (#3)
stodirascu/storyteller-brightcove-collector-swift: README.md

## Setup

Add Storyteller Brightcove Collector using SPM:

1. Add the package dependency: `https://github.com/getstoryteller/storyteller-brightcove-collector-swift`
1. Add the `StorytellerBrightcoveIntegration` target to your app target

To use the module, StorytellerSDK needs to be set to version `11.0.0` or higher.

> In the file where you setup the Storyteller SDK, import the added package, set up the module and set it on the `Storyteller` class:

Reviewer: ```suggestion
In the file where you setup the Storyteller SDK, import the added package, set up the module and set it upa as a `Storyteller` module:
```

```swift
import StorytellerBrightcoveIntegration

func initializeStoryteller() {
    let brightcoveConfiguration = StorytellerBrightcoveModuleConfiguration(
        account: "<account ID>",
        playerName: "Video Player", // optional
        source: "<source>",
        destination: "<destination>"
    )

    Storyteller.shared.modules.append(StorytellerBrightcoveModule(configuration: brightcoveConfiguration))

    // other Storyteller initialization code
}
```

> Code sample is also available in the [Showcase App](https://github.com/getstoryteller/storyteller-showcase-ios).

Reviewer: Think we can remove. Don't want too many links between repos that we'll have to maintain
